packages/tools/bcm2835-bootloader/scripts/update.sh

```diff
@@ -50,6 +50,12 @@
 #    sed -e "s,gpu_mem=100,gpu_mem=128,g" -i $BOOT_ROOT/config.txt
 #    sed -e "s,gpu_mem_256=100,# gpu_mem_256=128,g" -i $BOOT_ROOT/config.txt
 #    sed -e "s,gpu_mem_512=128,# gpu_mem_512=128,g" -i $BOOT_ROOT/config.txt
+  # Add distribution specific extra configuration file for the bootloader
+  fi
+
+# Add distro config file.
+  if [ -f $SYSTEM_ROOT/usr/share/bootloader/distroconfig.txt ]; then
+    cp $SYSTEM_ROOT/usr/share/bootloader/distroconfig.txt $BOOT_ROOT
```

Comment:
Almost there! :)

Can you make it `cp -p` like all the other copies in this file. Also, you need to rebase this file after #437.